renderlib/graphics/glsl/GLGuiShader.cpp

```diff
@@ -46,18 +46,19 @@ static const char* fragment_shader_text =
         vec4 result = Frag_color;
 
         // When drawing selection codes, everything is opaque.
-        if (picking == 1)
-            result.w = 1.0;
+        if (picking == 1) {
+          result.w = 1.0;
+        }
 
         // Gesture geometry handshake: any uv value below -64 means
         // no texture lookup. Check VertsCode::k_noTexture
-        if (picking == 0 && Frag_UV.s > -64)
-            result *= texture(Texture, Frag_UV.st);
+        if (picking == 0 && Frag_UV.x > -63.0) {
```

Comment:
Should this be `Frag_UV.x >= -63`?

Reply:
I'll add a comment here.  This was fixing a windows-only (and maybe nvidia-only) bug where the shader could not figure out whether `-64 > -64` .  The lines drawn were choppy.  So basically this is really Frag_UV.x >= (-64+epsilon)